src/account/AccountStorage.sol

```diff
@@ -68,6 +70,8 @@ struct AccountStorage {
     mapping(bytes4 => uint256) supportedIfaces;
     // Installed plugins capable of signature validation.
     EnumerableSet.Bytes32Set signatureValidations;
+    // Todo: merge this with other validation storage?
+    EnumerableSet.Bytes32Set defaultValidations;
```

Comment:
In the world where there is one default validation func, this field won't be necessary, right?

Reply:
It's transferred to be a property of validation functions in #64